pkg/config/reminder/metrics.go

```diff
@@ -0,0 +1,7 @@
+package reminder
+
+type MetricsConfig struct {
+	Enabled bool   `mapstructure:"enabled" default:"true"`
+	Host    string `mapstructure:"host" default:"127.0.0.1"`
```

Comment:
You may want to leave the address as `""`, so that Prometheus (for example) running in another network namespace can reach the metrics endpoint.